docs/deployment/index.md

```diff
@@ -17,14 +17,23 @@ _For more information about advanced configuration, read our documentation [here
 ```
 
 # Kubernetes
-We currently provide [a sample declaration](https://github.com/tomkerkhove/promitor/tree/master/deploy) which deploys all the required infrastructure on your Kubernetes cluster.
+We currently provide [a helm chart](https://github.com/tomkerkhove/promitor/tree/master/charts) which deploys all the required infrastructure on your Kubernetes cluster.
 
-Once downloaded, you can deploy it by running this command:
+You can then edit the values.yaml, or take a look at [this example yaml](https://github.com/tomkerkhove/promitor/blob/master/charts/local-values.yaml.example) to base your own on.
```

Comment:
```suggestion
You can then edit the `values.yaml`, or take a look at [our example configuration](https://github.com/tomkerkhove/promitor/blob/master/charts/local-values.yaml.example) to get you started.
```

Reply:
Made some slightly different edits, let me know if you want more changes

Reply:
👌